tests/test_hubspot_newfw_all_fields.py

```diff
@@ -0,0 +1,312 @@
+                                known_missing_keys.add(missing_key)
+                                del expected_record[missing_key]
+
+                        # NB : KNOWN_EXTRA_FIELDS is a dictionary of streams to fields that should not
+                        #      be replicated but are. See the variable declaration at top of file for linked BUGs.
+                        known_extra_keys = set()
+                        for extra_key in KNOWN_EXTRA_FIELDS.get(stream, set()):
+                            known_extra_keys.add(extra_key)
+
+                        # Verify the fields in our expected record match the fields in the corresponding replicated record
+                        expected_keys_adjusted = expected_keys.union(known_extra_keys)
+                        actual_keys_adjusted = actual_keys.union(known_missing_keys)
+
+                        # NB: The following woraround is for dynamic fields on the `deals` stream that we just can't track.
+                        #     At the time of implementation there is no customer feedback indicating that these dynamic fields
+                        #     would prove useful to an end user. The ones that we replicated with the test client are specific
+                        #     to our test data. We have determined that the filtering of these fields is an expected behavior.
+
+                        # deals workaround for 'property_hs_date_entered_<property>' fields
+                        bad_key_prefixes = {'property_hs_date_entered_', 'property_hs_date_exited_',
+                                            'property_hs_time_in'}
+                        bad_keys = set()
+                        for key in expected_keys_adjusted:
+                            for prefix in bad_key_prefixes:
+                                if key.startswith(prefix) and key not in actual_keys_adjusted:
+                                    bad_keys.add(key)
+                        for key in actual_keys_adjusted:
+                            for prefix in bad_key_prefixes:
+                                if key.startswith(prefix) and key not in expected_keys_adjusted:
+                                    bad_keys.add(key)
+                        for key in bad_keys:
+                            if key in expected_keys_adjusted:
+                                expected_keys_adjusted.remove(key)
+                            elif key in actual_keys_adjusted:
+                                actual_keys_adjusted.remove(key)
+
+                        self.assertSetEqual(expected_keys_adjusted, actual_keys_adjusted)
+
+    ##########################################################################
+    # Tests To Skip
+    ##########################################################################
+
+    @unittest.skip("Skip till all cards of missing fields are fixed. TDL-16145 ")
+    def test_all_fields_for_streams_are_replicated(self):
```

Comment:
Seems like the same test is listed twice and skipped here the second time.  Confusing

Reply:
I have removed the testname and added the code as a comment to avoid confusion